### DISCUSSION TITLE
Bank Holidays falling on weekends

### DISCUSSION THREAD

**Discussion by kieranbarlow** [spatie/holidays#50] [Q&A]
In the UK (England) any bank holiday falling on a Saturday or Sunday automatically shift to the next working day known as a substitute day.

Should we adjust for the bank holidays that fall on a weekend?

Example:

In 2023, New Year's Day, 1st of January landed on a Sunday so the bank holiday was on the 2nd January. Should New Year's Day return the 2023-12-01 or 2023-12-02.

https://www.gov.uk/bank-holidays

Thank you
Kieran

**Comment by jpgnz** [comment]
I think we need #123 answered on what the intent of this package is.
